### ISSUE TITLE
Error Under CP 3.0

### ISSUE THREAD

**Issue by jerryneedell** [adafruit/Adafruit_CircuitPython_SI7021#5]
On my metro M0_express, I am getting the following response when I try to initialize the si7021.
On the same board, same setupo it works under CP 2.x and Arduino
It also failed on a Metro M4 and nrf52 using 3.0 
This was using the Bundle relased on 20171223.
```Adafruit CircuitPython 3.0.0-alpha.1-16-g36ec29d on 2017-12-23; Adafruit Metro M0 Express with samd21g18
>>> 
>>> 
>>> import si7021_test
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "si7021_test.py", line 8, in <module>
  File "adafruit_si7021.py", line 79, in __init__
  File "adafruit_si7021.py", line 79, in __init__
  File "adafruit_si7021.py", line 75, in __init__
  File "adafruit_si7021.py", line 74, in __init__
  File "adafruit_bus_device/i2c_device.py", line 94, in write
OSError: 5
>>> 
```

**Comment by jerryneedell** [comment]
addressed with PR #6

**Comment by jerryneedell** [comment]
fixed with release 3.0.2
